src/components/SwipeableView/index.native.tsx

```diff
@@ -1,41 +1,34 @@
 import React, {useRef} from 'react';
 import {PanResponder, View} from 'react-native';
-import PropTypes from 'prop-types';
 import CONST from '../../CONST';
+import SwipeableViewProps from './types';
 
-const propTypes = {
-    children: PropTypes.element.isRequired,
-
-    /** Callback to fire when the user swipes down on the child content */
-    onSwipeDown: PropTypes.func.isRequired,
-};
-
-function SwipeableView(props) {
+function SwipeableView({children, onSwipeDown}: SwipeableViewProps) {
```

Comment:
It could make sense to define return type here and not let TS infer it.

Reason for that is that AFAIK now, TS will by defualt use type inferred from `index.tsx` and even if `index.native.tsx` returns something that does not conform to `index.tsx` return type, you won't know until something bad happens in runtime.

Reply:
According to TS guidelines we should define a common type only for props (return type for components is usually a `Element`/`ReactNode`). I'll discuss it with Fabio once he's back.

@kubabutkiewicz what's your opinion on this?

Reply:
I agree that this is good approach in general, but I am afraid it'll be one of places where we should consider an exception from that rule.

Reply:
Hmm , I am trying to imagine an example of the component which is returning something different from `Element`/`ReactNode` and nothing is coming to my mind. 
But if you know the example and in this case explicitly telling the return type would find the bug in build time , I am all in

Reply:
It still won't be bulletproof, but I can imagine such scenario, that someone, even by mistake, returns for example boolean from `SwipeableView/index.native.tsx` and since return type is inferred, it would be `ReactElement | boolean` or sth like that.
It would be fine in other cases, because that would be caught wherever this function will be used as JSX, but here TS will analyze only `SwipeableView/index.tsx` when using it, hence (I think!) no error will be shown.